### PR TITLE
develop: refine 2 docs

### PR DESCRIPTION
**Pull request by ran-huang**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

refine 2 docs:
- develop/dev-guide-sql-development-specification.md
- develop/dev-guide-unique-serial-number-generation.md

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-sql-development-specification.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 # SQL Development Specification
 
-This chapter will introduce some generalized development specifications for using SQL.
+This document introduces some general development specifications for using SQL.
 
-## Specification for table build and delete
+## Create and delete tables
 
 - Basic principle: the table is created under the premise of following the table naming specification, and it is recommended to add judgment logic to the build and delete table statements in the internal package of business applications to prevent abnormal interruption of business processes.
 - Details: `create table if not exists table_name` or `drop table if exists table_name` statements are recommended to add if judgments to avoid abnormal interruptions caused by SQL commands running abnormally on the application side.
```
Comment (ran-huang) [inline develop/dev-guide-sql-development-specification.md:13]: 这两段中文没读懂，在等 @Icemap 修改

File: develop/dev-guide-sql-development-specification.md [hunk 2]
```diff
@@ -1,25 +1,25 @@
 ---
 title: SQL Development Specification
```
Comment (hfxsd) [inline develop/dev-guide-sql-development-specification.md:2]:
```suggestion
title: SQL Development Specifications
```

File: develop/dev-guide-sql-development-specification.md [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
-summary: SQL Development Specification of TiDB.
+summary: Learn about the SQL development specification for TiDB.
```
Comment (hfxsd) [inline develop/dev-guide-sql-development-specification.md:3]:
```suggestion
summary: Learn about the SQL development specifications for TiDB.
```

File: develop/dev-guide-unique-serial-number-generation.md [hunk 4]
```diff
@@ -5,45 +5,47 @@ summary: Unique serial number generation solution for developers who generate th
...
-- worker node id: default `22` bits. Represents the machine id, usually obtained from a centralized ID generator when the application process is started. The default allocation policy is discard-as-you-go, and the process will reacquire a new worker node id on restart. 22 bits can support up to about `4.2 million` starts.
-- sequence: default `13` bits. The sequence of concurrency per second, `13` bits can support `8192` concurrency per second.
+- sign: Fixed length of 1 bit. Fixed to `0` to indicate that the generated ID is always a positive number.
+- delta seconds: 28 bits by default. The current time, presented as an incremental value in seconds relative to a preset time base (defaults to `2016-05-20`). 28 bits can support up to about 8.7 years.
+- worker node id: 22 bits by default. Represents the machine id, usually obtained from a centralized ID generator when the application process is started. Common centralized ID generator includes auto-increment columns and Zookeeper. The default allocation policy is discard-as-you-go, and the process re-acquires a new worker node id on restart. 22 bits can support up to about 4.2 million starts.
```
Comment (hfxsd) [inline develop/dev-guide-unique-serial-number-generation.md:36]:
```suggestion
- worker node id: 22 bits by default. Represents the machine ID, usually obtained from a centralized ID generator when the application process is started. Common centralized ID generators include auto-increment columns and Zookeeper. The default allocation policy is discard-as-you-go, and the process re-acquires a new worker node ID on restart. 22 bits can support up to about 4.2 million starts.
```

File: develop/dev-guide-unique-serial-number-generation.md [hunk 5]
```diff
@@ -5,45 +5,47 @@ summary: Unique serial number generation solution for developers who generate th
...
+| `STEP` | int(11) | The step, which indicates the length of each assigned segment. |
 
-The application gets a sequence number at each configured step and updates the database at the same time to persist the maximum value of the current sequence that has been allocated, then the sequence number processing and allocation can be completed in the application memory. After a number is exhausted, the application will get a new number, which effectively reduces the pressure on the database write. In practice, you can also adjust the step length to control the frequency of database record updates.
+Every time, the application gets a segment of sequence numbers at the configured step and updates the database at the same time to persist the maximum value of the current sequence that has been allocated. The processing and allocation of sequence numbers are completed in the application's memory. After a segment of sequence numbers is used up, the application gets a new segment of sequence numbers, which effectively alleviates the pressure on the database write. In practice, you can also adjust the step to control the frequency of database updates.
```
Comment (hfxsd) [inline develop/dev-guide-unique-serial-number-generation.md:49]:
```suggestion
Every time, the application gets a segment of sequence numbers at the configured step. It updates the database at the same time to persist the maximum value of the current sequence that has been allocated. The processing and allocation of sequence numbers are completed in the application's memory. After a segment of sequence numbers is used up, the application gets a new segment of sequence numbers, which effectively alleviates the pressure on the database write. In practice, you can also adjust the step to control the frequency of database updates.
```

File: develop/dev-guide-unique-serial-number-generation.md [hunk 6]
```diff
@@ -5,45 +5,47 @@ summary: Unique serial number generation solution for developers who generate th
...
 
-Finally, it should be noted that the IDs generated by the above two solutions are not random enough to be used as **primary keys** for TiDB tables directly. In practice, you can perform bit-reverse on the generated IDs to get a more random new ID.
+Finally, it should be noted that the IDs generated by the above two solutions are not random enough to be directly used as **primary keys** for TiDB tables. In practice, you can perform bit-reverse on the generated IDs to get a more random new ID.
```
Comment (hfxsd) [inline develop/dev-guide-unique-serial-number-generation.md:51]:
```suggestion
Finally, note that the IDs generated by the above two solutions are not random enough to be directly used as **primary keys** for TiDB tables. In practice, you can perform bit-reverse on the generated IDs to get more random new IDs.
```

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**